pre_commit/staged_files_only.py

```diff
@@ -47,7 +47,7 @@ def _unstaged_changes_cleared(patch_dir: str) -> Generator[None, None, None]:
         retcode=None,
     )
     if retcode and diff_stdout_binary.strip():
-        patch_filename = f'patch{int(time.time())}'
+        patch_filename = f'patch{os.getpid()}-{int(time.time())}'
```

Comment:
could you put the pid after the timestamp? then it will sort better

Reply:
good point - done!